dashboard_server.py

```
@@ -0,0 +1,615 @@
+                D = np.load(dist_path)
+                # Robustness: clip negatives and ensure symmetry
+                D = np.maximum(D, 0.0)
+                D = (D + D.T) / 2.0
+                S = 1.0 - (D / (np.max(D) + 1e-9))
+            else:
+                S = np.load(sim_path)
+                S = (S + S.T) / 2.0
+                D = 1.0 - S
+            with open(names_path, 'r', encoding='utf-8') as f:
+                names = json.load(f)
+        except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)